pyproject.toml

```diff
@@ -54,6 +51,13 @@ line-length = 119
 target-version = ['py37']
 include = '\.pyi?$'
 
+[tool.coverage.report]
+exclude_lines = [
+    "if TYPE_CHECKING:",
+    "pragma: no cover",
+    "raise NotImplementedError",
```

Comment:
Maybe this should be `raise NotImplementedError()` so that coverage does ignore it, and therefore pass the check? 🤔

Reply:
Good call, I added the `()`, but think I also needed to move things to `tox.ini`